sharedbuffers/mapped_struct.py

```diff
@@ -631,20 +638,20 @@ def __iter__(self):
             yield self[i]
 
     def __reversed__(self):
-        for i in reversed(xrange(len(self))):
+        for i in xrange(len(self) - 1, -1, -1):
```

Comment:
Since len() in Cython probably returns an unsigned type, better check that it isn't 0 before doing this.